src/compute/src/render/continual_task.rs

```diff
@@ -572,35 +573,29 @@ where
     output_stream.as_collection()
 }
 
-// TODO(ct2): Write this as a non-async operator.
 fn step_forward<G, D, R>(name: &str, input: Collection<G, D, R>) -> Collection<G, D, R>
```

Comment:
Please add a comment to describe what the function is supposed to do.

Reply:
Done!